zcash_client_sqlite/src/wallet/common.rs

```diff
@@ -158,7 +158,7 @@ where
                  INNER JOIN transactions
                     ON transactions.id_tx = {table_prefix}_received_notes.tx
                  WHERE {table_prefix}_received_notes.account_id = :account
-                 AND value >= 5000 -- FIXME #1016, allow selection of a dust inputs
+                 AND value > 5000 -- FIXME #1316, allow selection of dust inputs
```

Comment:
`check_for_uneconomic_inputs` treats an input as dust if it is less than or equal to the marginal fee. So to avoid the possibility that the input selection code in `propose_transaction` triggers a `DustInputs` error that it currently cannot handle for inputs of exactly the marginal fee, we should make this change as well until #1316 is fixed.

(I could make `check_for_uneconomic_inputs` only treat inputs of less than the marginal fee as dust, but I think that's not what we want.)

Reply:
Why not do what we discussed and change this to use `SUM(value - 5000)` as well?

Reply:
Because it's completely out of scope for this PR.